[2021-05-25 09:04:27,518] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: udac_example_dag.Run_data_quality_checks 2019-01-13T02:00:00+00:00 [queued]>
[2021-05-25 09:04:27,528] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: udac_example_dag.Run_data_quality_checks 2019-01-13T02:00:00+00:00 [queued]>
[2021-05-25 09:04:27,528] {taskinstance.py:879} INFO - 
--------------------------------------------------------------------------------
[2021-05-25 09:04:27,528] {taskinstance.py:880} INFO - Starting attempt 1 of 1
[2021-05-25 09:04:27,528] {taskinstance.py:881} INFO - 
--------------------------------------------------------------------------------
[2021-05-25 09:04:27,536] {taskinstance.py:900} INFO - Executing <Task(DataQualityOperator): Run_data_quality_checks> on 2019-01-13T02:00:00+00:00
[2021-05-25 09:04:27,539] {standard_task_runner.py:53} INFO - Started process 5372 to run task
[2021-05-25 09:04:27,597] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: udac_example_dag.Run_data_quality_checks 2019-01-13T02:00:00+00:00 [running]> Dylan.local
[2021-05-25 09:04:27,614] {data_quality.py:22} INFO - DataQualityOperator not implemented yet
[2021-05-25 09:04:27,617] {taskinstance.py:1065} INFO - Marking task as SUCCESS.dag_id=udac_example_dag, task_id=Run_data_quality_checks, execution_date=20190113T020000, start_date=20210525T160427, end_date=20210525T160427
[2021-05-25 09:04:37,504] {logging_mixin.py:112} INFO - [2021-05-25 09:04:37,503] {local_task_job.py:103} INFO - Task exited with return code 0
